changelog.md

```diff
@@ -1,71 +1,75 @@
 <a name="8.0.0"></a>
-(TODO)
 # 8.0.0 (2021-06-01)
 [Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v7.5.0...v8.0.0)
 
-~~ TODO: https://chromiumdash.appspot.com/schedule ~~
-We expect this release to ship in the DevTools of [Chrome XX](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+We expect this release to ship in the DevTools of [Chrome 93](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within a day!
 
-## Notable Changes
-~~ TODO: Move notable changes here ~~
+## Notable changes
+* The **Performance Category** had a number of scoring changes to align with other performance tools and to better reflect the state of the web:
+  - TODO: decide on FAQ incorporation and either replace these with that or flesh these out
+  - scoring: rebalance perf metric weightings ([#12577](https://github.com/GoogleChrome/lighthouse/pull/12577))
+  - scoring: update TBT score curve ([#12576](https://github.com/GoogleChrome/lighthouse/pull/12576))
+  - update cumulative-layout-shift ([#12554](https://github.com/GoogleChrome/lighthouse/pull/12554))
+  - update FCP score curve ([#12556](https://github.com/GoogleChrome/lighthouse/pull/12556))
+* The report includes a new metric filter. Pick a metric to focus on the opportunities and diagnostics most relevant to improving just that metric:
 
```

Comment:
Is it worth linking up to the score calc + include a screenshot of it for the perf category section?

Reply:
for fun i added that in https://github.com/GoogleChrome/lighthouse/pull/12597

Reply:
looks good!